api-samples/browsingData/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "BrowsingData API: Basics",
   "version": "1.2",
-  "description": "An example implementation of the chrome.browsingData API",
+  "description": "This sample demonstrates using chrome.contentSettings to display the settings of a given page in the extension's popup.",
```

Comment:
I think this might be intended to have the description from my previous comment? 😅